Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG PARENT_VERSION=2.3.0-node18.20.3
+ARG PARENT_VERSION=2.4.0-node18.20.4
```

Comment:
Are we not bumping the node version to 22

Reply:
Not in the old apps as that would involve a lot of rework as there will be breaking changes.  We could in the future but for now its mainly the patch release of node 18 and the update of the base image.